docsource/modules140-150.rst

```diff
@@ -640,11 +640,11 @@ Module coverage 14.0 -> 15.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | sale_timesheet                                  |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| sale_timesheet_edit                       |                      |Merged into sale_timesheet.                      |
+| |del| sale_timesheet_edit                       |Nothing to do         |Merged into sale_timesheet.                      |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |new| sale_timesheet_margin                     |                      |                                                 |
+| |new| sale_timesheet_margin                     |Nothing to do         |No analysis file. No DB layout changes.          |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| sale_timesheet_purchase                   |                      |                                                 |
+| |del| sale_timesheet_purchase                   |Nothing to do         |                                                 |
```

Comment:
This is not correct. It should be merged with something.

Reply:
Hi, sorry i have been out for a longtime and can't quite remember this, do you remember which module it should be merged with?

P/s: probably i will comeback for upgrade 15 -> 16 in june :v:

Reply:
Thank you, i'll keep in mind